app/src/test/java/com/movie/app/MovieRepositoryTest.kt

```diff
@@ -1,148 +1,142 @@
 package com.movie.app
 
-import com.movie.app.api.result.MoviesResult
-import com.movie.app.modules.Movie
 import com.movie.app.modules.MovieSearchFilter
 import com.movie.app.repositories.MovieDataSource
 import com.movie.app.repositories.MovieRepository
-import com.movie.app.repositories.remote.RemoteMovieRepository
-import com.movie.app.rx.TestHelper
-import com.nhaarman.mockito_kotlin.whenever
-import io.reactivex.Observable
-import io.reactivex.Observer
-import io.reactivex.observers.TestObserver
-import org.junit.Assert
+import com.movie.app.repositories.RemoteMovieDataSource
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.impl.annotations.MockK
+import kotlinx.coroutines.runBlocking
 import org.junit.Before
 import org.junit.Test
-import org.mockito.Mock
-import org.mockito.MockitoAnnotations
 import java.io.IOException
 
 class MovieRepositoryTest {
 
-    @Mock
-    private lateinit var localRep: MovieDataSource
-    @Mock
-    private lateinit var remoteRep: RemoteMovieRepository
-
+    @MockK
+    lateinit var localRep: MovieDataSource
+    @MockK
+    lateinit var remoteRep: RemoteMovieDataSource
     private lateinit var movieRep: MovieRepository
 
-    @Mock
+    @MockK
     private lateinit var networkException: IOException
-    @Mock
+    @MockK
     private lateinit var diskException: IOException
-    private lateinit var observer: Observer<MoviesResult>
 
     @Before
     fun setUp() {
-        MockitoAnnotations.initMocks(this)
+        MockKAnnotations.init(this, relaxUnitFun = true)
         movieRep = MovieRepository(localRep, remoteRep)
-        observer = TestHelper.mockObserver()
+        /*       MockitoAnnotations.initMocks(this)
+               observer = TestHelper.mockObserver()*/
```

Comment:
**Suggestion:** Remove commented out code to improve readability and maintainability.
```suggestion

```